docs/en/observability/ingest-metrics.asciidoc

```diff
@@ -117,17 +117,25 @@ include::{beats-repo-dir}/tab-widgets/start-widget.asciidoc[]
 // end::start-step[]
 
 [[view-metrics-kibana]]
-== Step 6: View your data in {kib}
+== Step 6: Confirm metrics are ingested
 
-To view the <<observability-ui,Observability Overview>> page:
+Let's confirm your data is correctly ingested to your cluster.
 
 . Launch {kib}:
 +
 --
 include::{beats-repo-dir}/tab-widgets/open-kibana-widget.asciidoc[]
 --
 
-. In the side navigation, select *Observability*, and click *Overview*. 
+. In the side navigation, select *{kib}*, and click *Discover*.
++
+. Select `metricbeat-*` as your index pattern. 
++
+Each document in the index that matches the `metricbeat-*` index pattern
+is displayed. By default, *Discover* shows data for the last 15 minutes.  If you have
+a time-based index, and no data displays, you might need to increase the time range.
```

Comment:
aren't metrics always time-based, though?

Reply:
That's true. I'll remove the sentence.